api/posters/statistics/send.spec.ts

```diff
@@ -80,17 +82,18 @@ describe('sendPosterStatistics', () => {
         .mockResolvedValueOnce()
       await sendPosterStatistics()
 
-      expect(sendEmailSpy).toHaveBeenCalledWith(
-        expect.stringContaining('<td>typescript</td><td>1</td>'),
-      )
+      const $ = cheerio.load(sendEmailSpy.mock.calls[0][0])
 
-      expect(sendEmailSpy).toHaveBeenCalledWith(
-        expect.stringContaining('<td>haskell</td><td>1</td>'),
-      )
+      const rows = $('tr')
+      expect(rows).toHaveLength(2)
 
-      expect(sendEmailSpy).not.toHaveBeenCalledWith(
-        expect.stringContaining('<td>golang</td><td>1</td>'),
-      )
+      const typescriptRowContent = rows.first().text()
+      expect(typescriptRowContent).toContain('typescript')
+      expect(typescriptRowContent).toContain('1')
+
+      const haskellRowContent = rows.last().text()
+      expect(haskellRowContent).toContain('haskell')
+      expect(haskellRowContent).toContain('1')
```

Comment:
I don't like having another dependency just for one test. I think we can use `toMatchInlineSnapshot()` and verify with a string that we contain this data. 

On the other hand, we can also use `JSDOM` and mount it. That way we can use `document` selectors in it. What do you think?

Reply:
Yeah I thought the same thing but when running tests with `document` a bunch of errors missing `jsom` :thinking: I tried to fix it but didn't want to spend to much time thinking of it just for this test, so found this "quick" but not ideal workaround as you mention

Reply:
Perhaps you can try adding this comment at the top of the test file 

```
/**
 * @jest-environment jsdom
 */
```

For more info check the [jest docs](https://jestjs.io/docs/configuration#testenvironment-string)